### PR TITLE
CI: update cilium-cli to v0.9.2

### PR DESCRIPTION
**Pull request by tklauser**
`cilium hubble enable` now correctly waits for Cilium status to be
ready. Thus, the extra `cilium status --wait` after enabling Hubble can
be avoided.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -37,6 +37,7 @@ addons:
     packages:
       - kernel-package
       - gnupg
+      - ipset
```
Comment (joestringer) [inline .travis.yml:40]: Why is this change necessary here?

**Review by joestringer — APPROVED** [review]
Looks good, except for one change that looks unrelated.

File: .travis.yml (hunk 1 above)
Comment (tklauser) [reply .travis.yml:40]: This is unrelated and I forgot to revert before creating the commit. Will drop that change.

**Review by nebril — APPROVED** [review]

File: .github/workflows/conformance-multicluster.yaml [hunk 2]
```diff
@@ -266,11 +266,6 @@ jobs:
           cilium hubble enable --context ${{ steps.contexts.outputs.context1 }} ${{ steps.vars.outputs.hubble_enable_defaults }}
           cilium hubble enable --context ${{ steps.contexts.outputs.context2 }} ${{ steps.vars.outputs.hubble_enable_defaults }} --relay=false
```
Comment (nebril) [inline .github/workflows/conformance-multicluster.yaml:267]: This slightly increase the test run, because first cluster hubble will wait for hubble to be ready and only then enable hubble in second cluster (and wait). Would be nice to parallelize this, but this is a nit.

Comment (tklauser) [reply .github/workflows/conformance-multicluster.yaml:267]: Good point. I guess we could run the first one with `--wait=false` to circumvent that and keep the `cilium status --wait` in this case.

Comment (tklauser) [reply .github/workflows/conformance-multicluster.yaml:267]: Done. Incremental diff:

```diff
diff --git a/.github/workflows/conformance-multicluster-v1.10.yaml b/.github/workflows/conformance-multicluster-v1.10.yaml
index db0f4e4fde9d..a0ff7ab91c8e 100644
--- a/.github/workflows/conformance-multicluster-v1.10.yaml
+++ b/.github/workflows/conformance-multicluster-v1.10.yaml
@@ -260,8 +260,9 @@ jobs:
 
       - name: Enable Relay
         run: |
-          cilium hubble enable --context ${{ steps.contexts.outputs.context1 }} ${{ steps.vars.outputs.hubble_enable_defaults }}
+          cilium hubble enable --context ${{ steps.contexts.outputs.context1 }} ${{ steps.vars.outputs.hubble_enable_defaults }} --wait=false
           cilium hubble enable --context ${{ steps.contexts.outputs.context2 }} ${{ steps.vars.outputs.hubble_enable_defaults }} --relay=false
+          cilium status --wait --context ${{ steps.contexts.outputs.context1 }}
 
       - name: Enable cluster mesh
         run: |
diff --git a/.github/workflows/conformance-multicluster.yaml b/.github/workflows/conformance-multicluster.yaml
index 0b405066c0a1..43384814917a 100644
--- a/.github/workflows/conformance-multicluster.yaml
+++ b/.github/workflows/conformance-multicluster.yaml
@@ -263,8 +263,9 @@ jobs:
 
       - name: Enable Relay
         run: |
-          cilium hubble enable --context ${{ steps.contexts.outputs.context1 }} ${{ steps.vars.outputs.hubble_enable_defaults }}
+          cilium hubble enable --context ${{ steps.contexts.outputs.context1 }} ${{ steps.vars.outputs.hubble_enable_defaults }} --wait=false
           cilium hubble enable --context ${{ steps.contexts.outputs.context2 }} ${{ steps.vars.outputs.hubble_enable_defaults }} --relay=false
+          cilium status --wait --context ${{ steps.contexts.outputs.context1 }}
 
       - name: Enable cluster mesh
         run: |
```

File: .travis.yml (hunk 1 above)
Comment (tklauser) [reply .travis.yml:40]: Turns out I forgot to push the actual change doing this :facepalm: sorry about that!

I can send a PR removing it again if needed. This was meant to silence some log warning related to missing `ipset` binary during privileged tests on Travis CI.

Comment (tklauser) [reply .travis.yml:40]: I've had to rebase #17687 which also changes `.travis.yaml`, so I've added a commit to that PR to drop `ipset` again.
